packages/server/graphql/types/SetPokerSpectatePayload.ts

```diff
@@ -21,6 +25,24 @@ export const SetPokerSpectateSuccess = new GraphQLObjectType<any, GQLContext>({
         const meetingMember = await dataLoader.get('meetingMembers').load(meetingMemberId)
         return meetingMember
       }
+    },
+    stages: {
+      type: new GraphQLList(new GraphQLNonNull(EstimateStage)),
```

Comment:
-1 `type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(EstimateStage)))`
always nice to guarantee that you'll get an empty array on the client so you don't have to check nullability before iterating over it.


+1 the type here is fixed to `EstimateStage` which means we don't need the `phaseType` in the resolver source